### DISCUSSION TITLE
How can I show an "info" icon next to a label in a form?

### DISCUSSION THREAD

**Discussion by thebravoman** [element-plus/element-plus#13174] [Q&A]
My goal is to be able to have an icon where the use could click and understand more about what this input field does. 

```
 <el-form-item label="Type of performance routing:">
                                <el-select
                                    v-model="performanceType"
                                    :value="performanceType"
                                    default-first-option
                                    :disabled="!editView"
                                    @change="savePerformanceType(priority)">
                                    <el-option
                                        v-for="item in performanceTypes"
                                        :key="item.value"
                                        :label="item.label"
                                        :value="item.value" />
                                </el-select>
                            </el-form-item>
```
Here I have a performanceType. But right after the text "Type of performance routing:" I would like to set a picture that says how and why this performance type should be set and what it does. 

Usually I would use a span with an icon, but if I put a span before the el-select it appears on top of the el-select

**Comment by ryuhangyeong** [comment]
https://element-plus.org/en-US/component/select.html#custom-template

is this what you want?

Recreate your situation using Element Plus Playground.

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgdmVyc2lvbiBhcyB2dWVWZXJzaW9uIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgdmVyc2lvbiBhcyBFcFZlcnNpb24gfSBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBFbGVtZW50UGx1cyB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuXG5jb25zdCBtc2cgPSByZWYoJ0hlbGxvIFdvcmxkIScpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8aDE+e3sgbXNnIH19PC9oMT5cbiAgPGVsLWlucHV0IHYtbW9kZWw9XCJtc2dcIiAvPlxuXG4gIDxwPlxuICAgIDxlbC1pY29uIGNvbG9yPVwidmFyKC0tZWwtY29sb3ItcHJpbWFyeSlcIj48RWxlbWVudFBsdXMgLz48L2VsLWljb24+XG4gICAgRWxlbWVudCBQbHVzIHt7IEVwVmVyc2lvbiB9fSArIFZ1ZSB7eyB2dWVWZXJzaW9uIH19XG4gIDwvcD5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

**Comment by thebravoman** [comment]
This is not what I want. I cannot recreate it with ELement Plus Playground because I can't find how to do this thing with element plus.

What I am looking for is 

```
Type of performance routing (i}: [.............] 
```

This is a label followed by an icon, followed by an input/select field. The user could hover over the (i) and understand what "type of performance routing" is and how it could help them. 

But I can't find a way to create the (i) inside of the label.

**Comment by markhermano** [comment]
@thebravoman 
This might be what you're after:

Use the slot label of the `el-form-item`
```html
<template #label>
  <span>Type of performance routing:</span>
  <span>
    <el-tooltip>
      <your-icon-here></your-icon-here>
    </el-tooltip>
  </span>
</template>
```

```html
<el-form-item>
  <template #label>
    <span>Type of performance routing:</span>
    <span>
      <el-tooltip>
        <your-icon-here></your-icon-here>
      </el-tooltip>
    </span>
  </template>
  <el-select
    v-model="performanceType"
    :value="performanceType"
    default-first-option
    :disabled="!editView"
    @change="savePerformanceType(priority)"
  >
    <el-option
      v-for="item in performanceTypes"
      :key="item.value"
      :label="item.label"
      :value="item.value"
    />
  </el-select>
</el-form-item>
```
